doc/source/api/index.rst

```diff
@@ -43,6 +43,14 @@ system for reads and variants.
 .. toctree::
    references
 
+Sequence Annotations
```

Comment:
@reece, see we didn't make this up:

http://www.nature.com/subjects/sequence-annotation
http://www.uniprot.org/help/sequence_annotation
http://ensemblgenomes.org/info/data/annotation
https://seqannotationstudio.codeplex.com/
https://en.wikipedia.org/wiki/Genome_project#Genome_annotation
http://www.amazon.com/Annotation-Chapman-Mathematical-Computational-Biology/dp/1439841179

Reece Hart <notifications@github.com> writes:
> In doc/source/api/index.rst:
> 
> > @@ -43,6 +43,14 @@ system for reads and variants.
> >  .. toctree::
> >     references
> >
> > +Sequence Annotations
> 
> Hmm. Okay, this is probably just means that you and I (and maybe the larger we)
> have different colloquial definitions of "annotations" and "features". There's
> probably no right or wrong here, but we should get some consensus on how the
> terms are used. (Ditto for the use of "allele" and "variant"
> semi-interchangeably elsewhere in the schema.)
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub*
>